frontend/src/static/js/v_zoom.js

```diff
@@ -51,18 +51,25 @@ window.vZoom = {
   watch: {
     info() {
       this.updateExpandedCommitMessagesCount();
+      this.setInfoHash();
```

Comment:
Added this as previously, opening new zoom tabs will not update the existing zoom hashes